data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			label main
+			`	You are eventually led into a very comfortable office where you are received by a mature looking Hai with the apropriate regalia of a diplomat. She holds out a paw to greet you, and says, "My name is Sayari. I was the envoy to the human government, a century ago."`
+			`	The present media storm is all the evidence you need to know the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them. If they're in on the secret, then it makes sense that they're probably still trying to work out a suspression strategy and have been too busy to send an envoy quicker than yourself.`
+			`	After a moment to let you ponder, she continues, "Our reports tell us that you are a well-known individual, respected both by your own government and abroad in the other human governments. Our reports also tell us that 'Hai' is quickly becoming a buzzword in your space. A few traders brought the news ahead of you."`
+			`	She pauses and looks at you seriously.`
+			`	"We think it would be prudent for you to represent your people as our point of contact. Would you accept such a role?"`
+			`	You give it a moment of thought.`
+			choice
+				`	"It certainly seems rather practical."`
+			`	"Fantastic," she says, though her tone is much more business like than the wording would imply, "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
+			choice
+				`	"Various? We only have two governments."`
+			`	"Oh," she says, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
+			`	You take the boxed item with you and an escort awaits to return you to your ship. As the door behind you closes you are sure you hear Sayari mutter to someone, "It was really much easier when they just had one government though, what a mess."`
+				accept
+	
+	
+mission "Hai Leaks Response 2"
+	landing
+	name "Collect Representatives"
+	description "Collect human government representatives from <stopovers> and take them to <destination>."
+	source "Earth"
+	stopover "Deep"
+	stopover "Foundry"
+	destination "Hai-home"
+	blocked "You have a mission here for the Hai, but you need three bunks free."
+	clearance
+	to offer
+		has "Hai Leaks Response 1: done"
+	passengers 3
+	on offer
+		event "pirates respond to leaks"
+		event "hai ladybug available"
+		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man with a firm handshake. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and the recent peace.`
```

Comment:
This is, I think, an area where I disagree somewhat with the strict dedication to duplicating logbook structure. There are occasional references to a characteristic in some logs and I'm in favour of adding slightly more rather than strictly avoiding them.

Reply:
While a fair point, I'd still say the amount of characterizing is a little too much. Maybe you could just remove `with a firm handshake`? If you prefer to keep it all in, then we'd be at an impasse, I guess. In that case, maybe we could wait for a third party with more experience than we have to weigh in? That might be making this a larger issue than it really is though. Not really sure. Your choice, I guess.

Reply:
I'm alright with taking that bit out.

Reply:
Updated the suggestion according to the discussion.